src/core/Object3D.js

```diff
@@ -702,6 +702,22 @@ Object3D.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 		}
 
+		if ( this.isScene ) {
+
+			if ( this.background && this.background.isColor ) {
+
+				object.background = this.background.getHex();
+
+			} else if ( this.background && this.background.isTexture ) {
+
+				object.background = serialize( meta.textures, this.background );
```

Comment:
Please use the approach similar to `Material.toJSON()` when serializing textures:
```js
object.background = this.background.toJSON( meta ).uuid;
````